### PR TITLE
Fixes for bugs found in 3.1 release candidate 3

### PR DESCRIPTION
**Pull request by pkmitre**

### CODE REVIEW

File: VRDR.CLI/Program.cs [hunk 1]
```diff
@@ -668,6 +668,67 @@ static int Main(string[] args)
                     Console.WriteLine($"{info.Field, -5} {info.Name,-15} {Truncate(info.Contents, 75), -75}: \"{field + "\"",-80}");
                 }
             }
+            else if (args.Length == 2 && args[0] == "extract")
+            {
+                BaseMessage message = BaseMessage.Parse(File.ReadAllText(args[1]));
+                switch(message)
+                {
+                    case DeathRecordSubmission submission:
```
Comment (hadleynet) [inline VRDR.CLI/Program.cs:676]: Does this also work for DeathRecordUpdate instances (a subclass of DeathRecordSubmission)?

File: VRDR.CLI/Program.cs [hunk 2]
```diff
@@ -668,6 +668,67 @@ static int Main(string[] args)
...
+                message.MessageSource = "http://mitre.org/vrdr";
+                Console.WriteLine(message.ToJSON());
+                return 0;
+            }
+            else if (args.Length == 2 && args[0] == "ack")
+            {
+                BaseMessage message = BaseMessage.Parse(File.ReadAllText(args[1]));
+                switch(message)
+                {
+                    case DeathRecordSubmission submission:
```
Comment (hadleynet) [inline VRDR.CLI/Program.cs:704]: same comment as above re DeathRecordUpdate

File: VRDR.CLI/Program.cs [hunk 3]
```diff
@@ -668,6 +668,67 @@ static int Main(string[] args)
...
                 }
             }
+            else if (args.Length == 2 && args[0] == "extract")
+            {
+                BaseMessage message = BaseMessage.Parse(File.ReadAllText(args[1]));
+                switch(message)
+                {
+                    case DeathRecordSubmission submission:
+                        var record = submission.DeathRecord;
+                        Console.WriteLine(record.ToJSON());
```
Comment (hadleynet) [inline VRDR.CLI/Program.cs:678]: Consider ToJSON(true) to get pretty printed output.

**Review by hadleynet — CHANGES_REQUESTED** [review]

File: VRDR.CLI/Program.cs (hunk 1 above)
Comment (pkmitre) [reply VRDR.CLI/Program.cs:676]: It does!

File: VRDR.CLI/Program.cs (hunk 3 above)
Comment (pkmitre) [reply VRDR.CLI/Program.cs:678]: Done for messages (where pretty printing is available)

File: VRDR.CLI/Program.cs (hunk 2 above)
Comment (pkmitre) [reply VRDR.CLI/Program.cs]: Removed this switch since it was actually dead code

**Pull request merged**